edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Kubernetes/EdgeOperator.cs

```diff
@@ -97,6 +103,23 @@ public class EdgeOperator : IKubernetesOperator, IRuntimeInfoProvider
 
                 ContractResolver = new CamelCasePropertyNamesContractResolver()
             };
+
+            if (!string.IsNullOrWhiteSpace(persistentVolumeName))
+            {
+                this.persistentVolumeName = KubeUtils.SanitizeK8sValue(persistentVolumeName);
+                this.persistentVolumeNameIsSc = false;
+            }
+            else if (!string.IsNullOrWhiteSpace(storageClassName))
+            {
+                this.persistentVolumeName = KubeUtils.SanitizeK8sValue(storageClassName);
+                this.persistentVolumeNameIsSc = true;
```

Comment:
This seems to suggest that if both SC and PV name are given, then the PV is considered to be the value that's in effect. Is this intentional? If yes, I'd consider putting a comment here explaining this detail.

Reply:
Also, it might be good to log a warning message to indicate that both an SC and a PV name were found and that we are using only the PV name to provision persistent volumes.